.vscode/settings.json

```diff
@@ -16,36 +24,14 @@
     "**/CVS": true
   },
   // Javascript
-  "javascript.validate.enable": false,
   "[javascript]": {
-    "editor.defaultFormatter": "numso.prettier-standard-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
@iwarner Why Typescript? Is this added intentionally?

Reply:
Not sure why that got added removed